reference/promotions.v3.yml

```
@@ -1642,6 +1718,54 @@ components:
               total: 5
               success: 3
               failed: 2
+
+    BulkCouponCodesResponse:
+      description: ''
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              data:
+                type: object
+                properties:
+                  created:
+                    type: string
+                    format: date-time
+                    description: The date and time when the codes were created.
+                  max_uses:
+                    type: integer
+                    description: The maximum number of times each code can be used.
+                  max_uses_per_customer:
+                    type: integer
+                    description: The maximum number of times each customer can use a code.
+                  batch_size:
+                    type: integer
+                    description: The number of codes generated in the batch.
+                  codes:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/BulkCouponCode'
+              meta:
+                type: object
+                properties: {}
+                description: 'Empty meta object, which may be used at a later time.'
+          examples:
+            example-1:
+              value:
+                data:
+                  max_uses: 10
+                  max_uses_per_customer: 5
+                  created: '2019-01-20T22:00:00+00:00'
+                  "batch_size": 5
+                  codes:
+                    - code: "WFYUYQM1W1ZYGK7S"
```

probably we don't need double quotes
```suggestion
                    - code: WFYUYQM1W1ZYGK7S
```